src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -537,9 +537,8 @@ protected void markRadioButtonAsChecked(WebElement radioButton) {
     public void selectDropdownByVisibleValue(WebElement element, String value) {
         Select select = new Select(element);
         select.selectByVisibleText(value);
-        String selectedVisibleValue = select.getFirstSelectedOption().getText();
+        String selectedVisibleValue = select.getFirstSelectedOption().getText().trim();
```

Comment:
The Chrome Driver doesn't auto-trim the text, i.e things like this:
```html
<select>
    ABCD
</select>
```
Will return `ABCD` with all the leading and trailing spaces.